src/modules/tooltip.coffee

```diff
@@ -37,6 +37,8 @@ class Tooltip
     @container.style.left = Tooltip.HIDE_MARGIN
```

Comment:
To take this one step further, perhaps this style could be moved into a stylesheet as well?